composer.json

```
@@ -28,7 +28,7 @@
     "drupal/seven": "^1.0",
     "drush/drush": "^12.2",
     "skilldlabs/drupal-cleanup": "^1",
-    "skilldlabs/druxxy": "dev-main"
+    "skilldlabs/druxxy": "^1"
```

```suggestion
    "skilldlabs/druxxy": "^1.1"
```
let's use for precise